io-engine/src/constants.rs

```diff
@@ -2,10 +2,10 @@
 pub const NEXUS_CAS_DRIVER: &str = "Nexus_CAS_Driver";
 
 /// Data plane's NVMe controller model ID.
-pub const NVME_CONTROLLER_MODEL_ID: &str = "Mayastor NVMe controller";
+pub const NVME_CONTROLLER_MODEL_ID: &str = "Other NVMe controller";
```

Comment:
Other --> Mayastor ?

Reply:
hum... these were just here for testing...